flowsa/methods/flowbysectoractivitysets/NEI_Nonpoint_2017_asets.csv

```diff
@@ -1,594 +1,594 @@
+activity_set,name,note
+direct_allocation,2801500111,Agriculture - Field Burning
+direct_allocation,2801500112,Agriculture - Field Burning
+direct_allocation,2801500120,Agriculture - Field Burning
+direct_allocation,2801500130,Agriculture - Field Burning
+direct_allocation,2801500141,Agriculture - Field Burning
+direct_allocation,2801500142,Agriculture - Field Burning
+direct_allocation,2801500150,Agriculture - Field Burning
+direct_allocation,2801500160,Agriculture - Field Burning
+direct_allocation,2801500170,Agriculture - Field Burning
+direct_allocation,2801500171,Agriculture - Field Burning
+direct_allocation,2801500181,Agriculture - Field Burning
+direct_allocation,2801500182,Agriculture - Field Burning
+direct_allocation,2801500191,Agriculture - Field Burning
+direct_allocation,2801500192,Agriculture - Field Burning
+direct_allocation,2801500201,Agriculture - Field Burning
+direct_allocation,2801500202,Agriculture - Field Burning
+direct_allocation,2801500210,Agriculture - Field Burning
+direct_allocation,2801500220,Agriculture - Field Burning
+direct_allocation,2801500230,Agriculture - Field Burning
+direct_allocation,2801500240,Agriculture - Field Burning
+direct_allocation,2801500250,Agriculture - Field Burning
+direct_allocation,2801500261,Agriculture - Field Burning
+direct_allocation,2801500262,Agriculture - Field Burning
+direct_allocation,2801500310,Agriculture - Field Burning
+direct_allocation,2801500320,Agriculture - Field Burning
+direct_allocation,2801500330,Agriculture - Field Burning
+direct_allocation,2801500340,Agriculture - Field Burning
+direct_allocation,2801500350,Agriculture - Field Burning
+direct_allocation,2801500370,Agriculture - Field Burning
+direct_allocation,2801500380,Agriculture - Field Burning
+direct_allocation,2801500390,Agriculture - Field Burning
+direct_allocation,2801500400,Agriculture - Field Burning
+direct_allocation,2801500410,Agriculture - Field Burning
+direct_allocation,2801500420,Agriculture - Field Burning
+direct_allocation,2801500430,Agriculture - Field Burning
+direct_allocation,2801500440,Agriculture - Field Burning
+direct_allocation,2801500450,Agriculture - Field Burning
+cropland,2801000000,Agriculture - Crops
+cropland,2801000001,Agriculture - Crops
+cropland,2801000002,Agriculture - Crops
+cropland,2801000003,Agriculture - Crops
+cropland,2801000004,Agriculture - Crops
+cropland,2801000005,Agriculture - Crops
+cropland,2801000006,Agriculture - Crops
+cropland,2801000007,Agriculture - Crops
+cropland,2801000008,Agriculture - Crops
+cropland,2801500000,Agriculture - Field Burning
+cropland,2801500151,Agriculture - Field Burning
+cropland,2801500152,Agriculture - Field Burning
+cropland,2801500161,Agriculture - Field Burning
+cropland,2801500263,Agriculture - Field Burning
+cropland,2801500264,Agriculture - Field Burning
+cropland,2801500300,Agriculture - Field Burning
+cropland,2801500360,Agriculture - Field Burning
+industry_combustion_a,2102001000,Fuel Combustion - Industry
```

Comment:
for consistency with the other updates, maybe we should just go ahead and label these with the fuel type like `industry_combustion_coal`

Reply:
That's actually what I started doing, but some of the fuel types, like "Hydrocarbon gas liquids, excluding natural gasoline" are pretty long, and using only a portion of the name, like `hydrocarbon_liquids`, implies something that isn't true. Thoughts?

Reply:
Nevermind, just pushed an update with better names.

Reply:
Good point!